libs/isograph-react/src/core/cache.ts

```diff
@@ -562,28 +638,40 @@ function normalizeNetworkResponseObject(
   variables: Variables,
   index: number | null,
   nestedRefetchQueries: RefetchQueryNormalizationArtifactWrapper[],
-  mutableEncounteredIds: Set<DataId>,
+  mutableEncounteredIds: EncounteredIds,
+  queryType: Link,
 ): DataId /* The id of the modified or newly created item */ {
   const newStoreRecordId = getDataIdOfNetworkResponse(
     targetParentRecordLink,
     networkResponseData,
     astNode,
     variables,
     index,
+    queryType,
   );
+  const __typename =
```

Comment:
since this is the only place that we use `.queryType`, let's get rid of it and just hard code it here, along with 

```
// TODO do not hard code this value here
```

I'm willing to bet that no one will have a nested query type, while *also* renaming the query type, before we have a proper solution. I'll sketch out the proper solution in #209